mod/groups/views/default/object/discussion_reply.php

```diff
@@ -38,11 +38,19 @@
 
 $subtitle = "$poster_text $date";
 
-$content = elgg_view('output/longtext', array(
-	'value' => $reply->description,
-	'class' => 'elgg-inner',
-	'data-role' => 'discussion-reply-text',
-));
+if (elgg_in_context('activity')) {
+	$content = '<div class="elgg-output elgg-inner" data-role="comment-text">';
+	$content .= elgg_view('output/text', array(
+		'value' => elgg_get_excerpt($reply->description)
+	));
+	$content .= '</div>';
```

Comment:
nitpick: I'd prefer having opening and closing tag on the same row for readability.